Gems/EMotionFX/Code/Source/Integration/System/SystemComponent.cpp

```diff
@@ -626,7 +626,7 @@ namespace EMotionFX
                 // Check if we are in game mode.
                 IEditor* editor = nullptr;
                 AzToolsFramework::EditorRequestBus::BroadcastResult(editor, &AzToolsFramework::EditorRequests::GetEditor);
-                inGameMode = editor->IsInGameMode();
+                inGameMode = editor && editor->IsInGameMode();
```

Comment:
Can it be that it is being run not from editor?  in this case this indication will not be correct

Reply:
Correct.. This crashed launching the material editor.
Your suggestion reduces to inGameMode = !editor || editor->IsInGameMode();
That also works and preserves the default.